lib/ddtrace/version.rb

```diff
@@ -6,8 +6,9 @@ module VERSION
     MINOR = 11
     PATCH = 0
     PRE = 'beta1'
+    BUILD = nil
```

Comment:
I recommend leaving a comment here saying that PRE and BUILD can be dynamically changed during our gem build GH Action.

Reply:
Done